collections/Add_A/MS_Add_A_263.xml

```diff
@@ -37,13 +41,39 @@
                      <idno type="SCN">29383</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>Guillelmus de Tocco, <title>Ystoria sancti Thome de Aquino</title>, with offices for Thomas Aquinas and Corpus Christi. A, B Germany, 14th century (after 1318)</head>
                <physDesc>
-                  <p>Composite: two parts</p>
+                  <objectDesc form="codex">
+                     <supportDesc>
+                        <extent> iii + 48 + 16 + i leaves
+                           <dimensions type="leaf" unit="mm">
+                              <height min="240" max="245">240–5</height>
+                              <width max="175" min="170">170–5</width>
+                           </dimensions></extent>
+                        <foliation> 1–64 in late 19th-century purple pencil, i–iii and 65v in 20th-century pencil</foliation>
+                     </supportDesc>
+                  </objectDesc>
+                  <bindingDesc>
+                     <binding contemporary="true">
+                        <p>Contemporary binding. Sewn on four pairs of twisted parchment bands, laced and pegged separately into horizontal channels in wood (beech?) boards, covered with undecorated tawed(?) skin, overlapping the edges of the leaves; two strap-and-pin clasps (closing from the back to the front board); the front cover inscribed with a shelfmark in bold 15th-century characters ‘Y.22.’; the top of the spine inscribed ‘Legen<lb/>da S. | Thom<lb/>as [ … ] | &amp; | S. Aug. | S. Ambr.’ (worn; readings uncertain; cf. fols. 49–54).
+                        </p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
+               <history>
+                  <provenance>Doubtless written for a Dominican house in Germany, and perhaps still there in the 15th century.</provenance>
```

Comment:
'Doubtless' seems here and below to be indicating a measure of uncertainty, which should be made clearer.